content/ops/code-review.md

```diff
@@ -0,0 +1,58 @@
+---
+menu:
+  main:
+    parent: operations
+title: Code Review
+---
+
+This document describes how the 18F cloud.gov team has agreed to work together
+with regard to Git, Github and facilitating changes to the repositories they administer.
+
+## Workflow
+
+The most common workflow is:
+
+1. A change is initiatied and discussed via a [Trello card](https://trello.com/b/ChGzyepo/paas-dev).
+1. A Pull Request (PR) is created on Github.
+1. The PR is reviewed by someone other than the committer. Pairing via screen-sharing
+is encouraged and qualifies as a review.
+1. The reviewer merges the PR.
+
+## Forking vs Branching
+
+The team prefers forking.
+
+The master branch on the canonical repository is what gets deployed to production.
+If there is a staging environment available for a given repository, the PR should be
+created against the staging branch. Otherwise, the PR should be created against the master
+branch on the canonical repository.
+
+The rationale for preferring forking is that all contributors work the same way,
+regardless of whether or not they may commit directly to the canonical repository.
+
+## Squashing commits
+
+[Squashing commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits)
+is allowed but discouraged, except in rare instances.
+
+## Rebase or Merge
+
+The team prefers [rebasing over merging](https://www.atlassian.com/git/tutorials/merging-vs-rebasing/).
```

Comment:
@mogul you are right. We didn't make it as clear but the idea is to keep the merges limited to Pull Requests. That way we can keep the history clean when it is indeed merged.